charts/datadog/templates/cluster-agent-deployment.yaml

```diff
@@ -183,6 +183,8 @@ spec:
           - name: DD_ADMISSION_CONTROLLER_INJECT_CONFIG_LOCAL_SERVICE_NAME
             value: {{ template "localService.name" . }}
           {{- end }}
+          - name: DD_ADMISSION_CONTROLLER_FAILURE_POLICY
+            value: {{ .Values.clusterAgent.admission_controller.failurePolicy | quote }}
```

Comment:
```suggestion
            value: {{ .Values.clusterAgent.admissionController.failurePolicy | quote }}
```

Reply:
Just noticed this in the failed check.  Thanks for pointing it out :)